tests/v2_validation/cattlevalidationtest/core/test_webhook.py

```diff
@@ -0,0 +1,116 @@
+from common_fixtures import *  # NOQA
+import requests
+from requests.auth import AuthBase
+import json
+
+access_key = os.environ.get("CATTLE_ACCESS_KEY", 'admin')
```

Comment:
You might be able to get the keys off of the client. Can you see if those are available to you instead of getting them from env vars?